python/ray/serve/_private/default_impl.py

```diff
@@ -204,3 +209,25 @@ def get_proxy_handle(endpoint: DeploymentID, info: EndpointInfo):
         )
 
     return handle.options(stream=not info.app_is_cross_language)
+
+
+def get_controller(http_options, grpc_options, global_logging_config):
```

Comment:
This is starting the controller actor. Can we either rename it to start_controller, or keep as is but only return the `ray.remote(ServeController)` and start it in the call site? (would prefer second one so it is consistent with other functions in this file)